Source/Base/Base/Math/Spline.h

```diff
@@ -0,0 +1,138 @@
+#pragma once
+
+#include "Base/Types.h"
+
+#include <glm/glm.hpp>
+#include <glm/gtc/matrix_transform.hpp>
+
+namespace Spline
+{
+    struct InterpolatedStorage
+    {
+    protected:
```

Comment:
Variables at the end of the struct (Fix this for all structs/classes in this file)

struct X
{
-- Public Struct defintions
-- Public Class defintions
-- Private Struct defintions
-- Private Class defintions

-- Public Member Functions
-- Private Member Functions

-- Public/Private Variables
}